ci-operator/config/openshift/metallb-operator/openshift-metallb-operator-main.yaml

```
@@ -99,6 +103,72 @@ tests:
     workflow: generic-claim
+- always_run: false
+  as: tls-scanner
+  optional: true
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.22"
+  optional: true
```

YAMLlint (1.37.1)

[error] duplication of key "optional" in mapping

(key-duplicates)